SimpleTDDCsharpTest/PalindromeCheckerTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+
+using Xunit;
+using FluentAssertions;
```

Comment:
can we put repeated using blocks in global.cs file?

Reply:
yes done this change.